tests/integration-test/src/tests/utils.rs

```diff
@@ -460,6 +464,66 @@ pub fn add_kretprobe(
     Ok(prog_id)
 }
 
+/// Install a fentry program with bpfman
+pub fn add_fentry(load_type: &LoadType, image_url: &str, file_path: &str) -> Result<String> {
```

Comment:
Nit but can you just collapse these into a single function with a boolean toggle or something? We already have so much duplicated code here. :/

Reply:
Done